src/base/TilesRendererBase.js

```diff
@@ -185,7 +225,10 @@ export class TilesRendererBase {
 
 		}
 
+		// Expected to be set during calculateError()
+		tile.__distanceFromCamera = Infinity;
```

Comment:
Should this also set `tile.__error = Infinity` here?

Reply:
Yeah I think `tile.__error = Infinity` is fine to initialize to considering it's reset every update.